### PR TITLE
Added Bootstrap 3 and Foundation 6 Autocomplete

### PR DESCRIPTION
**Pull request by webchun**

### CODE REVIEW

**Comment by wbond** [comment]
Is this compatible with Sublime Text 2? If not, the `sublime_text` selector in this PR should be `">=3000"`. If yes, change the repo description.

Also, you need to create a [semver](http://semver.org) tag on the repo before anyone will be able to install it.

**Comment by webchun** [comment]
Hi @wbond sorry for the delay. Yes it supports Sublime Text 2. I have also published the release with semver tag. Thanks

**Comment by wbond** [comment]
The Foundation 6 Autocomplete needs to be using tags also. Currently you have submitted it as a branch.

**Comment by webchun** [comment]
I think I have published it in the correct way : https://github.com/webchun/foundation-6-sublime-autocomplete/tags 
Am I missed something?

File: repository/f.json [hunk 1]
```diff
@@ -1204,6 +1204,17 @@
 			]
 		},
 		{
+			"name": "Foundation 6 Autocomplete",
+			"details": "https://github.com/webchun/foundation-6-sublime-autocomplete",
+			"labels": ["auto-complete"],
+			"releases": [
+				{
+					"sublime_text": "*",
+					"branch": "master"
```
Comment (wbond) [inline repository/f.json]: This line here needs to be changed to `"tags": true`

**Comment by webchun** [comment]
Done thanks

**Pull request merged**